subworkflows/nf-core/bam_trim_primers_ivar/main.nf

```diff
@@ -0,0 +1,44 @@
+include { IVAR_TRIM               } from '../../../modules/nf-core/ivar/trim/main'
+include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+
+workflow BAM_TRIM_PRIMERS_IVAR {
+    take:
+    ch_bam   // channel: [ val(meta), path(bam), path(bai) ]
+    ch_bed   // channel: [ path(bed) ]
+    ch_fasta // channel: [ path(fasta) ]
+
+    main:
+    ch_versions = Channel.empty()
+
+    //
+    // iVar trim primers
+    //
+    IVAR_TRIM (
+        ch_bam,
+        ch_bed
+    )
+    ch_versions = ch_versions.mix(IVAR_TRIM.out.versions.first())
+
+    //
+    // Sort, index BAM file and run samtools stats, flagstat and idxstats
+    //
+    BAM_SORT_STATS_SAMTOOLS (
+        IVAR_TRIM.out.bam,
+        ch_fasta
+    )
+
+    ch_versions = ch_versions.mix(BAM_SORT_STATS_SAMTOOLS.out.versions.first())
+
+    emit:
+    bam_orig = IVAR_TRIM.out.bam                    // channel: [ val(meta), bam   ]
```

Comment:
```suggestion
    bam_orig = IVAR_TRIM.out.bam                    // channel: [ val(meta), path(bam) ]
```

Can you add paths to all the files in the comments (see above for an example :))